charts/newrelic-infra-operator/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: A Helm chart to deploy the New Relic Infrastructure Kubernetes Operator.
 name: newrelic-infra-operator
-version: 0.5.0
-appVersion: 0.5.0
+version: 0.6.0
+appVersion: 0.6.0
```

Comment:
Until the `0.6.0` version is deployed an orphan chart is going to be available.

It is a NIT that does not block the release because we are not using `nri-bundle` yet but is something we have to care in the future ;)

Reply:
you mean [this](https://github.com/newrelic/newrelic-infra-operator/releases/tag/v0.6.0) one? Or do you mean something else?

Reply:
I thought you were going to release it again because of this change: https://github.com/newrelic/newrelic-infra-operator/pull/157/files#diff-8df2af49373ef833809e6615c110f9c7ac8fdacf8dc6f6c0215ba948e6666046R62-R64

What is that spurious change?

Reply:
that is a change to make a flaky test less flaky